firefox-ios/Client/Frontend/Components/BaseAlphaStackView.swift

```diff
@@ -111,13 +117,11 @@ class BaseAlphaStackView: UIStackView, AlphaDimmable, ThemeApplicable {
 
     func applyTheme(theme: Theme) {
         let isVersionLayout = toolbarLayoutType == .version1 || toolbarLayoutType == .version2
-        let color: UIColor = if isClearBackground {
-            .clear
-        } else {
-            isVersionLayout ? theme.colors.layer3 : theme.colors.layer1
-        }
-        backgroundColor = color
-        keyboardSpacer?.backgroundColor = color
-        insetSpacer?.backgroundColor = color
+        let color: UIColor = isVersionLayout ? theme.colors.layer3 : theme.colors.layer1
+        let backgroundAlpha = ToolbarHelper().backgroundAlpha()
```

Comment:
nit: This is not huge, but I find it a bit strange we need an "empty" instance to call some methods on `ToolbarHelper`, which make me wonder if those properties/methods can be static.